src/layers.cpp

```diff
@@ -44,6 +45,8 @@ RangeArr<std::string, 1, maxEvents> NewEvent;
 RangeArrI<int, 1, maxEvents, 0> newEventDelay;
 int newEventNum = 0;
 
+static std::set<std::string> recentlyTriggeredEvents;
```

Comment:
This should be easy to update to std::set<eventindex_t> when we begin merging the devel branch improvements. But it may be even better to just add a "WasRecentlyTriggered" member to Event_t since events are now referenced exclusively through their indices, not their names. (For the Luna, we will probably want to use a Name -> index hash map.)

Reply:
Right, this thing is just an initial experiment, and I think, it can be improved later easily. So indeed, it needs to convert the raw name into the index number easily, that can be done on the script parsing stage easily (we can store all "MyString" fields at the string collections on the parsing stage, and just re-use them, easily).